app/models/spell.rb

```diff
@@ -0,0 +1,16 @@
+class Spell < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+  validates :level, numericality: {greater_than_or_equal_to: 0, less_than_or_equal_to: 9}
+  validates :range, presence: true
+  validates :school, presence: true
+  validates :casting_time, presence: true
+  validates :duration, presence: true
+  validates :description_short, presence: true
+  validates :description_long, presence: true
+
+  has_and_belongs_to_many :spell_lists
+
+  def multiple_labels
```

Comment:
probably could use a better intention-revealing name, maybe like `to_simpleform_label` ?

Reply:
Actually I've missed the fact that simple_form allows me to skip the `:label_method` altogether if I name this method `:to_label` in my model.